rules/do-you-use-loggermessage-in-net.md/rule.md

```diff
@@ -0,0 +1,114 @@
+---
+type: rule
+title: Do you use LoggerMessage in .NET?
```

Comment:
Hi @tomek-i 

- [ ] 1. Please add SEO Description (**Tip:** Look on other rules and use ChatGPT ;))